City-of-Helsinki/helsinki-paatokset
# PP-310: Add latest news-block

Added latest new -block to sidebar for imported articles -pages. 

To test:
- Run `drush cim -y; drush cr`
- Run the migration: `drush mim paatokset_news --update`
- Check that you have [content](https://helsinki-paatokset.docker.so/fi/admin/content?title=&type=imported_article&status=All&langcode=All)
- Open one of the imported articles. Latest news-block should be displayed in the sidebar on the right side. Links should take to the article pages.
